syntax = "proto3";

package PbCan;

message CanFrequency
{
	uint64 timestamp = 1;
	uint64 id = 2;
	string name = 3;
	uint64 data = 4;
	uint64 frequency = 5;
}

message CanFrequencies
{
	map<uint64, CanFrequency> frequencies = 1;
}

message CanNetworksFrequencies
{
	uint64 timestamp = 1;
	map<string, CanFrequencies> networks = 2;
}